core/spec/lib/search/base_spec.rb

```diff
@@ -37,26 +37,28 @@
   end
 
   context "when ascend_by_master_price scope is included in the initalization params" do
-    let(:params) { { search:{
-                     ascend_by_master_price: nil }
-                     }
-                   }
+    let(:params) {
+      { search: {
+                     ascend_by_master_price: nil
+} }
+    }
```

Comment:
I'm not sure I'd call this style "fixed". Probably fine to do something more like this, same as below.

```suggestion
    let(:params) {
      { search: { ascend_by_master_price: nil } }
    }
```

It's preferred if you squash these changes into the original commit to avoid cluttering up the history with reformatting commits.

Reply:
Sorry @jarednorman, you are right! I have only launched rubocop to shut up Hound 😅
Now I'll squash all changes in one commit with the fix and you'll can see the CI green with a well formated code. Thanks and sorry for this mess 😄